### DISCUSSION TITLE
Is there an RT limit?

### DISCUSSION THREAD

**Discussion by jdekarske** [lnccbrown/HSSM#361] [Q&A] [answered]
I'm working on a modeling task with a longer reaction time and I'm running into a weird problem with what seems like a hard limit on RT. Using a default DDM example with fake params
```
v_true, a_true, z_true, t_true = [0.001, 4, 0.5, 1]
```
![image](https://github.com/lnccbrown/HSSM/assets/20480052/3b893adb-e313-4371-b834-4860eefdd8a2)

I can add a large `max_t` to the ssms simulator for the sample data, but not for the posterior sampling.
![image](https://github.com/lnccbrown/HSSM/assets/20480052/7fa0f073-067c-477c-a00e-ce63fcaf09bb)

I would love any help :)

**Comment by AlexanderFengler** [comment, marked as answer]
Hey Jason, 

few things: 

1. We will look into the posterior predictive to fix the hard limit
2. Fitting to data with such long rts is, for now, going to be possible only for the analytical likelihood with the `ddm` model or the `blackbox`, `full_ddm` model. All the network based likelihoods (`approx_differentiable`) are trained only on rts between `[0,20]` seconds. 
3. Networks can in principle be trained on longer rts so this is not a structural hard limit, but we haven't done that for two reasons. First, the usual application context just doesn't demand it, so this actually never came up so far. Second, to put some limits on runtime for generating training data. 

Best,
Alex

**Reply by jdekarske** [reply]
Awesome, thank you so much!
